api/userPlants/userPlants-model.js

```diff
@@ -9,15 +9,17 @@ function findById(id) {
 }
 
 function findUserPlantsByPlantsID(user_plant_id){
-    return db('user_plants')
-        .where({user_plant_id})
+    return db("user_plants as up")
+        .join("species as sp", "up.species_id", "sp.species_id")
+        .join("water_schedule as ws", "sp.water_id", "ws.water_id")
+        .where("up.user_plant_id", user_plant_id)
+        .select("up.user_plant_id", "up.plant_nickname", "up.plant_location", "up.notes", "up.water_day", "sp.species_id", "sp.plant_name", "sp.plant_scientific_name", "sp.plant_image", "ws.water_schedule")
         .first();
 }
 
 async function addPlant(newPlant){
     const [user_plant_id] = await db('user_plants')
-        .returning('user_plant_id')
```

Comment:
a mini question about this, so we dont want to return the plant data after insert? guessing it will automatically be generated in the the DB and show in the UI?